cvat/apps/dataset_manager/tests/assets/tasks.json

```diff
@@ -54,6 +54,190 @@
       }
     ]
   },
+  "icdar_localization_and_recognition": {
+    "name": "icdar localization/recogntion task",
+    "overlap": 0,
+    "segment_size": 100,
+    "owner_id": 1,
+    "assignee_id": 2,
+    "labels": [
+      {
+        "name": "icdar",
+        "attributes": [
+          {
+            "name": "text",
+            "mutable": false,
+            "input_type": "text",
+            "values": ["word_1", "word_2", "word_3"]
+          }
+        ]
+      }
+    ]
+  },
+  "icdar_segmentation": {
+    "name": "icdar segmentation task",
+    "overlap": 0,
+    "segment_size": 100,
+    "owner_id": 1,
+    "assignee_id": 2,
+    "labels": [
+      {
+        "name": "icdar",
+        "attributes": [
+          {
+            "name": "text",
+            "mutable": false,
+            "input_type": "text",
+            "values": ["word_1", "word_2", "word_3"]
+          },
+          {
+            "name": "index",
+            "mutable": false,
+            "input_type": "number",
+            "values": ["0", "1", "2"]
+          },
+          {
+            "name": "color",
+            "mutable": false,
+            "input_type": "text",
+            "values": ["100 110 240", "10 15 20", "120 128 64"]
+          },
+          {
+            "name": "center",
+            "mutable": false,
+            "input_type": "text",
+            "values": ["1 2", "2 4", "10 45"]
+          }
+        ]
+      }
+    ]
+  },
+  "market1501": {
+    "name": "market1501 task",
+    "overlap": 0,
+    "segment_size": 100,
+    "owner_id": 1,
+    "assignee_id": 2,
+    "labels": [
+      {
+        "name": "market-1501",
+        "attributes": [
+          {
+            "name": "query",
+            "mutable": false,
+            "input_type": "select",
+            "values": ["True", "False"]
+          },
+          {
+            "name": "camera_id",
+            "mutable": false,
+            "input_type": "number",
+            "values": ["1", "2", "3", "4", "5", "6"]
```

Comment:
@MashaSS , could you please help me to recall? Why do we have a list of values for "text" and "number"?

Reply:
Values for "number" define range of possible values (min, max, step). Values for "text" are used in requests but don't define any value restrictions.

Reply:
Values for text should be text. Do you agree?

Reply:
Yes, could you please clarify what test values should be changed? Attributes `color` and `center` in `icdar_segmentation` task?